### PR TITLE
Overwrite the existing templates to the new syntax

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

**Comment by gela-d** [comment]
@tadatuta @tavriaforever

File: src/blocks/common.blocks/post/_view/post_view_list-item.bemtree [hunk 1]
```diff
@@ -1,6 +1,6 @@
-block post, mod view list-item {
+block('post').mod('view', 'list-item')(
 
-    content: {
+    content()(function () {
         var node = this.ctx.node,
             source = node.source && node.source[this.data.lang];
```
Comment (tadatuta) [inline src/blocks/common.blocks/post/_view/post_view_list-item.bemtree:5]: the same issue with apply syntax as in previous PR

Comment (gela-d) [inline src/blocks/common.blocks/post/_view/post_view_list-item.bemtree:5]: @tadatuta done

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**